### ISSUE TITLE
Heading always rendered as <h1>

### ISSUE THREAD

**Issue by tegan-rbi** [webstudio-is/webstudio-designer#6]
Right now, the heading element is always rendered as an h1 tag.

There really shouldn't be more than one h1 tag on a page. The user should be able to specify a level 1 - 6 which results in an h1-h6 tag. Otherwise, maybe programmatically derive heading levels based on nested sections or something like that.

**Comment by kof** [comment]
Absolutely. I wonder if we should make it a generic text input or be specific what can be entered for each type of component. For example Box component could be potentially anything (maybe just not an input)
